### PR TITLE
Cypress/e2e: Fixed test cases

### PR DESCRIPTION
**Pull request by furqanmlk**
```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/tests/integration/bot_accounts/tags_spec.js [hunk 1]
```diff
@@ -48,7 +48,9 @@ describe('Bot tags', () => {
                 await client.pinPost(postId);
 
                 cy.visit(`/${team.name}/channels/${channel.name}`);
-                cy.clickPostDotMenu(postId);
```
Comment (M-ZubairAhmed) [inline e2e/cypress/tests/integration/bot_accounts/tags_spec.js:51]: can we update the `clickPostDotMenu` itself?

Comment (furqanmlk) [reply e2e/cypress/tests/integration/bot_accounts/tags_spec.js:51]: > can we update the `clickPostDotMenu` itself?

Thanks @M-ZubairAhmed, for the review,  `clickPostDotMenu` function is still required for features or other test cases. These changes are only to trigger the emoji menu using mouseover

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by saturninoabril** [comment]
@furqanmlk Please do the changes at mattermost-server and submit a PR there. Then this can be closed.

**Comment by M-ZubairAhmed** [comment]
Hello there! Could you please open the PR in the mattermost-server as part of the monorepo? I will be closing this PR for now. Thank you very much for your understanding and cooperation in this matter.

**Pull request closed without merging**